src/awkward/_errors.py

```diff
@@ -287,6 +309,7 @@ def wrap_error(
 
     Wrap the given exception, instantiating it if needed, to ensure meaningful error messages.
     """
+    return exception
     if isinstance(exception, type) and issubclass(exception, Exception):
         try:
             exception = exception()
```

Comment:
Is `wrap_error` now dead code? (It was always L3; we can remove it.)

It looks like this is making it inert by returning the argument and then making everything that comes after it dead. Is that debugging code?

Reply:
Yes, this was a discussion pr. Must have slipped on the draft button!